Source/Events.Processing/Streams/Partitioned/FailingPartitions.cs

```diff
@@ -44,12 +44,12 @@ public class FailingPartitions : IFailingPartitions
         /// <inheritdoc/>
         public async Task<IStreamProcessorState> AddFailingPartitionFor(IStreamProcessorId streamProcessorId, StreamProcessorState oldState, StreamPosition failedPosition, PartitionId partition, DateTimeOffset retryTime, string reason, CancellationToken cancellationToken)
         {
-            var failingPartition = new FailingPartitionState(failedPosition, retryTime, reason, 1);
+            var failingPartition = new FailingPartitionState(failedPosition, retryTime, reason, 1, DateTimeOffset.UtcNow);
             var failingPartitions = new Dictionary<PartitionId, FailingPartitionState>(oldState.FailingPartitions)
                 {
                     [partition] = failingPartition
                 };
-            var newState = new StreamProcessorState(failedPosition + 1, failingPartitions);
+            var newState = new StreamProcessorState(failedPosition + 1, failingPartitions, DateTimeOffset.MinValue);
```

Comment:
This sets the LastSuccessfullyPRocessed to MinValue. That's not right. It should be the previous value (oldState.LastSuccessfullyProcessed)